cisc/commands.go

```diff
@@ -359,5 +359,82 @@ func getCommands() cli.Commands {
 				},
 			},
 		},
+
+		{
+			Name:    "cert",
+			Aliases: []string{"c"},
+			Usage:   "create and use links with admin privileges",
+			Subcommands: cli.Commands{
+				{
+					Name:      "request",
+					Aliases:   []string{"q"},
+					Usage:     "request a certificate to letsencrypt and store it to the skipchain",
+					ArgsUsage: "name of the directory",
+					Action:    certRequest,
+					Flags: []cli.Flag{
+						cli.StringFlag{
+							Name:  "directory, dir",
```

Comment:
Replace `directory, dir` with `cert-name` - `Give name for the certificate file`